region/norfair/west.json

```diff
@@ -1668,51 +1668,134 @@
                       ]},
-                    {"heatFrames": 300},
-                      {"or": [
-                        "h_heatProof",
-                          {"enemyDamage": {
-                            "enemy": "Sova",
-                            "type": "contact",
-                            "hits": 1
-                          }}
+                      "canWalljump",
+                      "canSpringBallJumpMidAir",
+                      {"and": [
+                        "h_canJumpIntoIBJ",
+                        {"heatFrames": 240}
+                      ]},
+                      {"and": [
+                        "h_canJumpIntoIBJ",
+                        "h_canDoubleBombJump",
+                        {"heatFrames": 120}
+                      ]},
+                      {"and": [
+                        "h_canDoubleBombJump",
+                        {"heatFrames": 300}
                       ]}
+                    ]},
+                    {"heatFrames": 260},
+                    {"enemyDamage": {
+                      "enemy": "Sova",
+                      "type": "contact",
+                      "hits": 1
+                    }}
                   ],
-                  "note": "The Sovas should be killed if you are heat proof."
+                  "devNote": "Base IBJ should kill the Sovas or go around."
                 },
                 {
-                  "name": "Power Bombs",
+                  "name": "Kill Sovas",
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    "h_canUsePowerBombs",
-                      {"or": [
-                        "HiJump",
+                    "Morph",
+                    {"or": [
+                      "HiJump",
+                      {"and": [
                         "SpaceJump",
-                        "canWalljump"
+                        {"heatFrames": 40}
                       ]},
-                    {"heatFrames": 300}
+                      "canWalljump",
+                      "canSpringBallJumpMidAir",
+                      {"and": [
+                        "h_canIBJ",
+                        {"heatFrames": 900}
+                      ]},
+                      {"and": [
+                        "h_canJumpIntoIBJ",
+                        {"heatFrames": 240}
+                      ]},
+                      {"and": [
+                        "h_canJumpIntoIBJ",
+                        "h_canDoubleBombJump",
+                        {"heatFrames": 120}
+                      ]},
+                      {"and": [
+                        "h_canDoubleBombJump",
+                        {"heatFrames": 300}
+                      ]}
+                    ]},
+                    {"or": [
+                      "h_heatProof",
+                      {"and": [
+                        "Wave",
+                        {"heatFrames": 360}
```

Comment:
360 frames in addition to the 260 below?

Reply:
Yeah I couldn't cleanly kill the sovas.  Maybe there's a better way than what I was trying.  This option is fairly pointless as it is.

Reply:
Jumping and shooting works ok.  Diagonal shots can work but take either additional movement tech or more time to clear all the Sovas, comparatively.